sci-geosciences/gmt/gmt-4.5.18-r1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="
+	mirror://gmt/${P}-src.tar.bz2
+	gmttria? ( mirror://gmt/${P}-non-gpl-src.tar.bz2 )"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples debug +gdal gmttria +gshhg htmldoc +metric mex +netcdf octave postscript tutorial"
+
+RDEPEND="
+	!sci-biology/probcons
+	gdal? ( sci-libs/gdal )
+	gshhg? ( sci-geosciences/gshhg-data )
+	netcdf? ( >=sci-libs/netcdf-4.1 )
+	octave? ( sci-mathematics/octave )"
+DEPEND="${RDEPEND}"
+
+# mex can use matlab too which i can't test
+REQUIRED_USE="
+	mex? ( octave )
+"
+
+# hand written make files that are not parallel safe
+MAKEOPTS+=" -j1"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.5.9-no-strip.patch
+	"${FILESDIR}"/${PN}-4.5.6-respect-ldflags.patch
+	)
+
+AUTOTOOLS_IN_SOURCE_BUILD=1
+
+src_configure() {
+	econf \
```

Comment:
Please put all options into `myconf=()` array and pass it as `"${myconf[@]}"` afterwards to avoid the backslash hell.

Reply:
Done